pipeline-runner/.Rprofile

```diff
@@ -1,2 +1,3 @@
 source("renv/activate.R")
+options(Seurat.object.assay.version = "v5")
```

Comment:
Does seurat v5 behave nicely with older seurat objects? I did not have time to play around with it yet.

Does it upgrade old objects to v5? I'm concerned with the worker (did not look at that pr yet), since changing the way we access the slots/layers feels like it would break functionality.

Reply:
Not entirely, no. All accessor functions of the form `scdata[['RNA']]@slot` needed to be updated to `scdata[['RNA']]$layer` which works with both Seurat v3 and v5